.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
   - make deps
 
 script:
-  - make test
+  - sudo make test
```

Comment:
I don't think we're doing this anywhere else. Is this supported by Travis?

Reply:
Bugger, left in from some testing